diff --git a/platform/widget/windows/nsFilePicker.cpp b/platform/widget/windows/nsFilePicker.cpp
index 59ae152ec..8dc603abc 100644
--- a/platform/widget/windows/nsFilePicker.cpp
+++ b/platform/widget/windows/nsFilePicker.cpp
@@ -35,6 +35,11 @@ using mozilla::mscom::EnsureMTA;
 using mozilla::UniquePtr;
 using namespace mozilla::widget;

+
+#ifndef FOS_OKBUTTONNEEDSINTERACTION
+#define FOS_OKBUTTONNEEDSINTERACTION 0x00200000
+#endif
+
 char16_t *nsFilePicker::mLastUsedUnicodeDirectory;
 char nsFilePicker::mLastUsedDirectory[MAX_PATH+1] = { 0 };

